light/lightchain.go

```diff
@@ -302,7 +302,7 @@ func (lc *LightChain) GetBlockByNumber(ctx context.Context, number uint64) (*typ
 // Stop stops the blockchain service. If any imports are currently in progress
 // it will abort them using the procInterrupt.
 func (lc *LightChain) Stop() {
-	if !atomic.CompareAndSwapInt32(&lc.running, 0, 1) {
+	if !lc.running.CompareAndSwap(false, true) {
```

Comment:
This variable name is weird. Perhaps stopped is more suitable.

Reply:
Thanks for the reminder, it has been modified.